code/modules/modular_computers/computers/machinery/modular_console.dm

```diff
@@ -13,7 +13,7 @@
 	base_idle_power_usage = 100
 	base_active_power_usage = 500
 	max_hardware_size = 3
-	steel_sheet_cost = 20
+	steel_sheet_cost = 10
```

Comment:
`steel_sheet_cost` is using for building but ignored in dismantle?

Reply:
It should. And somehow it's still using old system. I will look at that later.